components/core/ApplicationUserControls.js

```diff
@@ -5,7 +5,7 @@ import * as SVG from "~/common/svg";
 import {
   TooltipWrapper,
   dispatchCustomEvent,
-  PopoverNavigation,
+  PopoverNavigation
```

Comment:
We support trailing comma in prettier so maybe fix this one later

Reply:
Also this file shouldn't be in your PR whats going on.

Reply:
Yea I was looking at it to make sure my code was structured correctly, and then couldnt remvoe it from the pr. (I think I clicked save and prettier made some change.)